src/zarr/core/group.py

```diff
@@ -697,6 +702,99 @@ def from_dict(
             store_path=store_path,
         )
 
+    async def copy_to(
+        self,
+        store: StoreLike,
+        *,
+        path: str | None = None,
```

Comment:
what does it mean when path is None? is it meaningfully different from defaulting to the empty string?